naming/program/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "spl-naming"
```

Comment:
WDYT about calling this the:
```suggestion
name = "spl-nam-service"
```
and then moving `naming/*` into `/name-service/*`?

Reply:
sounds good!